dev/local-environment/configurations/cardano/cardano-node-2/keys/pledge

```diff
@@ -0,0 +1 @@
+1000000000000
```

Comment:
Are pledge and port files used anywhere? I can see `cat` for them, but maybe I've missed it. If they are not used, please don't commit them into repo.